tests/data/test_daily_bars.py

```diff
@@ -119,6 +119,8 @@
 
 TEST_QUERY_ASSETS = EQUITY_INFO.index
 
+HOLES = {3: (Timestamp('2015-06-17', tz='UTC'),)}
```

Comment:
Can we add a note on the description for this equity saying that it has a hole on this day?)